gemma-core/src/main/java/ubic/gemma/model/common/quantitationtype/ScaleType.java

```diff
@@ -38,6 +38,10 @@ public enum ScaleType {
      * An unscaled measurement is one that has no inherent scale; e.g., a categorial value.
      */
     UNSCALED,
+    /**
+     * Constrained to be a value between 0 and 1.
+     */
+    PERCENT1,
```

Comment:
We need to introduce this new scale type for data ranging in [0, 1] to complement PERCENT which ranges from [0, 100].